pkg/k8shandler/kibana/consolelink.go

```diff
@@ -0,0 +1,50 @@
+package kibana
+
+import (
+	consolev1 "github.com/openshift/api/console/v1"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func NewConsoleLink(name, namespace, href string) *consolev1.ConsoleLink {
+	return &consolev1.ConsoleLink{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Spec: consolev1.ConsoleLinkSpec{
+			Location: consolev1.ApplicationMenu,
+			Link: consolev1.Link{
+				Text: "Logging",
+				Href: href,
+			},
+			ApplicationMenu: &consolev1.ApplicationMenuSpec{
+				Section: "Monitoring",
```

Comment:
Is this section agreed upon?

Reply:
Based on: https://github.com/openshift/console/blob/master/frontend/public/actions/features.ts#L202